test/extended/util/util.go

```diff
@@ -269,7 +269,14 @@ func LoginToOcCluster(options []string) error {
 		ocLoginAsAdminCommand = ocLoginAsAdminCommand + " " + option
 	}
 
-	return ExecuteCommand(ocLoginAsAdminCommand)
+	err = ExecuteCommand(ocLoginAsAdminCommand)
```

Comment:
I think the code you add is similar to what `return ExecuteCommand(ocLoginAsAdminCommand, "succeeds")`  does?

Reply:
Thanks for pointing this out. I have re-commit the code change.